common/models/user.js

```diff
@@ -673,6 +673,38 @@ module.exports = function(User) {
       next();
     });
 
+    // Delete old sessions once email is updated
+    UserModel.observe('before save', function beforeEmailUpdate(ctx, next) {
+      if (ctx.isNewInstance) return next();
+      if (!ctx.where && !ctx.instance) return next();
+      var where = ctx.where || { id: ctx.instance.id };
+      ctx.Model.find({ where: where }, function(err, userInstances) {
+        if (err) return next(err);
+        ctx.hookState.originalUserData = userInstances.map(function(u) {
+          return { id: u.id, email: u.email };
+        });
+        next();
+      });
+    });
+    UserModel.observe('after save', function afterEmailUpdate(ctx, next) {
```

Comment:
Please add an empty line between `UserModel.observe` blocks to visually delimit them.

```js
UserModel.observe('before save', .
});

UserModel.observe('after save', ...
});

```